install/installer/pkg/components/ws-manager-mk2/role.go

```diff
@@ -69,6 +69,28 @@ func role(ctx *common.RenderContext) ([]runtime.Object, error) {
 						"update",
 					},
 				},
+				{
+					APIGroups: []string{"workspace.gitpod.io"},
+					Resources: []string{"snapshots"},
+					Verbs: []string{
+						"create",
+						"delete",
+						"get",
+						"list",
+						"patch",
+						"update",
+						"watch",
+					},
+				},
+				{
+					APIGroups: []string{"workspace.gitpod.io"},
+					Resources: []string{"snapshots/status"},
+					Verbs: []string{
+						"get",
+						"patch",
+						"update",
```

Comment:
Naive question: Do ws-manager-mk2 and ws-daemon both need patch and update for snapshots/status? I thought we were trying to consolidate update verbs to single sources of truth.

Reply:
Updated scope of permissions